Client/clientGameConnection.py

```diff
@@ -68,7 +79,6 @@ def remove_item_to_send(self, item_dto:ItemDto):
 
     def push_item_to_process(self, item_dto: ItemDto) -> None:
         self._items_to_process.append(item_dto)
-
-
-def print_item_dto(itemDto: ItemDto) -> None:
-    print(f"{WWR.item_name_dict[itemDto.itemId]} was found in world {itemDto.sourcePlayerWorldId}")
+    
+    async def print_item_dto(self, itemDto: ItemDto) -> None:
```

Comment:
I thought we had talked about this, but it might have gotten burried by everything else.

Feel free to move the string provided in `print_item_dto` as a function to the `itemDto` object. I'd name it something like `get_simple_output`. Then we can do `await self.log(itemDto.get_simple_output())` instead.